.vscode/settings.json

```diff
@@ -1,3 +1,20 @@
 {
```

Comment:
Did you mean to add your vscode settings?

Reply:
yea, I meant to save those as local settings. I removed them.